### PR TITLE
Update to Rails ujs readme

### PR DESCRIPTION
**Pull request by djforth**
### Summary

As Mentioned in #28992 An update to Rails ujs doc for user adding the rails-ujs through yarn/npm

### CODE REVIEW

**Comment by djforth** [comment]
Confused why travis failed? I only updated a readme? Am I missing something?

File: actionview/app/assets/javascripts/README.md [hunk 1]
```diff
@@ -36,6 +36,16 @@ Require `rails-ujs` into your application.js manifest.
 //= require rails-ujs
 ```
 
+Usage with yarn/webpacker
+------------
+
+Add the following to your main js file.
```
Comment (maclover7) [inline actionview/app/assets/javascripts/README.md]: `js` --> `JS`

File: actionview/app/assets/javascripts/README.md [hunk 2]
```diff
@@ -36,6 +36,16 @@ Require `rails-ujs` into your application.js manifest.
 //= require rails-ujs
 ```
 
+Usage with yarn/webpacker
```
Comment (maclover7) [inline actionview/app/assets/javascripts/README.md]: `Usage with npm package` might be more general?

**Review by maclover7 — CHANGES_REQUESTED** [review]
Left some small comments :)

File: actionview/app/assets/javascripts/README.md (hunk 2 above)
Comment (djforth) [inline actionview/app/assets/javascripts/README.md]: Sure will add, although thought it was important to mention yarn and webpacker as that's the recommended way now isn't it?.

Comment (djforth) [inline actionview/app/assets/javascripts/README.md]: Sorry I think I miss read it.  Can change it to  node_modules instead pf npm package, but do you think that may be confusing when you mention installing through Yarn previously in the readme?

I could update Installation using Yarn to include npm install instructions too then that might make it a little clearer maybe?

**Comment by maclover7** [comment]
@djforth Can you squash down to one commit

**Comment by djforth** [comment]
Really sorry, something gone a bit crazy with my git when I try to rebase to squash the commits it tries to squash a whole bunch of commits that are nothing to do with me.  

I've tried tried everything else I can think off to fix it, but I think the best solution as the change is so small is to kill the pull request, re-fork then re-add the update then issue another pull request.  Unless you have any better solutions or rather I didn't do that?

**Comment by rafaelfranca** [comment]
I merged manually at 51f912f1f3 and backported in d660b27e16. Thanks

**Comment by djforth** [comment]
Ok thanks

**Pull request closed without merging**